plugin/src/main/resources/io/jenkins/plugins/casc/ConfigurationAsCode/index.jelly

```diff
@@ -54,7 +54,7 @@
     <h2>${%Reference}</h2>
     <dt>
       <dl><a href="reference">${%Documentation}</a></dl>
-      <dl><a href="v2Schema">${%JSON schema}</a></dl>
+      <dl><a href="Schema">${%JSON schema}</a></dl>
```

Comment:
does `schema` work?

Reply:
I think it doesn't matter it still converts to small case

Reply:
should be lower then
```suggestion
      <dl><a href="schema">${%JSON schema}</a></dl>
```